src/Bicep.Cli.E2eTests/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {},
+  "devDependencies": {
```

Comment:
I think you'll need to add this path to https://github.com/Azure/bicep/blob/main/.github/dependabot.yml

Reply:
Good to know! I added it to the dependabot file.